update.php

```diff
@@ -886,6 +895,7 @@ function update_1102() {
 
 
 function update_1103() {
+/// @TODO Comented out:
```

Comment:
Fixed tpzo ... ;-) but the original committer is @friendica some how? commit id b795ce4f